blog-article.html

```diff
@@ -1,23 +1,23 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style-blogpage.css">
     <link rel="stylesheet" type="text/css" href="https://necolas.github.io/normalize.css">
```

Comment:
@Alinchik1 let's put "normalize.css" file in our project, like any other css files. You can take it from here:
https://github.com/necolas/normalize.css/blob/master/normalize.css

Reply:
added to all html files